src/database/migrations/20200312122638-create-request.js

```diff
@@ -49,6 +49,18 @@ module.exports = {
       allowNull: false,
       type: Sequelize.BOOLEAN,
     },
+    handledBy: {
+      allowNull: true,
+      type: Sequelize.INTEGER,
+      references: {
+        model: 'users',
+        key: 'id',
+      },
+    },
+    handlerNames: {
+      allowNull: true,
+      type: Sequelize.STRING,
+    },
```

Comment:
I think you don't need to add this field since you already referenced the handler in `handledBy` field